documentation/userguide/docs/worksheets.md

```diff
@@ -55,4 +55,54 @@ delete buttons).
     for more information on AWS Athena SQL syntax.
 
 
+## :material-new-box: **Experimental Features: GenAI Powered Worksheets**
+
+As part of data.all >= v2.7 we introduced support for generative AI powered worksheet features. These features include both:
+
+1. Natural Language Querying (NLQ) of Structured Data
+2. Text Document Analysis of Unstructured Data
+
+These features are optionally enabled/disabled via feature flags specified in data.all's configuration.
+
+The More details on how to use each of these features are below.
+
+### Natural Language Querying (NLQ) of Structured Data
+
+data.all offers a NLQ feature to significantly redeuce the barrier for non-technical business users who need to quickly and easily query data to make informed decisions, as mastering these technical skills can be time-consuming.
+
+The NLQ feature will take a user prompt and select number of tables and generate a ready to run SQL statement that data.all user's can execute against the data they have access to in data.all's Worksheets module.
```

Comment:
strange sentence: Given a prompt and a selection of tables, data.all NLQ will generate the corresponding SQL statement....